src/TestSuite/IntegrationTestTrait.php

```diff
@@ -1227,9 +1227,9 @@ public function assertFileResponse(string $expected, string $message = ''): void
      * Inspect controller to extract possible causes of the failed assertion
      *
      * @param string $message Original message to use as a base
-     * @return null|string
+     * @return string
      */
-    protected function extractVerboseMessage($message = null)
+    protected function extractVerboseMessage(string $message = ''): string
```

Comment:
In this case we dont even need the default value `''` IMO. A message has to be passed for this method to make sense (concat operation).